plenum/test/node_catchup/catchup_req/test_catchup_with_one_slow_node.py

```diff
@@ -75,11 +77,15 @@ def test_catchup_with_one_slow_node(tdir, tconf,
 
     # Delay CatchupRep messages on Alpha
     with delay_rules(rest_nodes[0].nodeIbStasher, cqDelay()):
-        with delay_rules(lagging_node.nodeIbStasher, cs_delay()):
+        with delay_rules(lagging_node.nodeIbStasher, cpDelay()):
             looper.add(lagging_node)
             txnPoolNodeSet[-1] = lagging_node
             looper.run(checkNodesConnected(txnPoolNodeSet))
+            # wait till we got consistency proofs from all nodes
+            looper.run(
+                eventually(lambda: assertExp(lagging_node.nodeIbStasher.num_of_stashed(ConsistencyProof) >= 3), retryWait=1,
+                           timeout=60))
 
-            waitNodeDataEquality(looper, *txnPoolNodeSet, customTimeout=120,
-                         exclude_from_check=['check_last_ordered_3pc_backup'])
-            assert len(log_re_ask) - old_re_ask_count == 2  # for audit and domain ledgers
+        waitNodeDataEquality(looper, *txnPoolNodeSet, customTimeout=120,
```

Comment:
Why move out of consistency proof delayer?

Reply:
If we don't move out, then we can not finish the view change, can't we?